update release cadence message (#42358)
dotnet/docs: includes/net-framework-future.md

> [!NOTE]

> > .NET Framework is serviced independently from Windows updates with security and reliability bug fixes.  The servicing schedule generally releases security updates quarterly. .NET Framework will continue to be included with Windows, with no plans to remove it. You don't need to migrate your .NET Framework apps, but for new development, use [.NET 8 or later](~/docs/core/introduction.md).

Reviewer: ```suggestion
> .NET Framework is serviced independently from Windows updates with security and reliability bug fixes. In general, security updates are released quarterly. .NET Framework will continue to be included with Windows, with no plans to remove it. You don't need to migrate your .NET Framework apps, but for new development, use [.NET 8 or later](~/docs/core/introduction.md).

```
